src/api/app/models/status_message.rb

```diff
@@ -8,6 +8,8 @@ class StatusMessage < ApplicationRecord
   scope :announcements, -> { order('created_at DESC').where(severity: 'announcement') }
   scope :for_current_user, -> { where(communication_scope: communication_scopes_for_current_user) }
   scope :newest, -> { order('created_at DESC') }
+  scope :for_severity, ->(severity) { where(severity: severity) if severity.present? }
```

Comment:
While these scopes are only used once in this PR, we believe they will be useful in the future, so we added them.